src/ansys/pyensight/core/session.py

```diff
@@ -972,6 +972,7 @@ def load_data(
         reader_options: Optional[dict] = None,
         new_case: bool = False,
         representation: str = "3D_feature_2D_full",
+        monitor_new_steps: bool = False,
```

Comment:
Not in the docstrings?

Reply:
I don't know why it's there. I am fixing it

Reply:
I would not doc it in this PR as you are changing the API a bit in the next PR.

Reply:
it shouldn't have been there at all, I am fixing it